### DISCUSSION TITLE
The icon is not display in the nav bar

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#12570] [Help] [answered]
Hi Team

I want to display an icon in my nav bar in React JS, but it is not displayed. My image is in the public folder. How can I solve this problem?

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/a2090cc4-b72c-4c14-9658-7e1597766f6f)

  

my code is: -

import React from 'react'

const nav = () => {
  return (

    <div className="header_color bg-center bg-no-repeat bg-cover w-full ">
        <nav className="flex  items-center justify-center ">
           <div className="w-48 ">
              <img src='YTmp32-logo.png' className='w-full h-10' />
           </div>

           <div className="w-2/4 ">
              <ul className=" text-xl flex p-4 justify-center gap-5">
                 <li className="text-white hover:text-indigo-200 ">
                    <a href="#">Home</a>
                 </li>
                 <li className="text-white hover:text-indigo-200">
                    <a href="#">Guide</a>
                 </li>
                 <li className="text-white hover:text-indigo-200">
                    <a href="#">About Us</a>
                 </li>
                 <li className="text-white hover:text-indigo-200">
                    <a href="#">Contact Us</a>
                 </li>
              </ul>
           </div>
        </nav>
     </div>
  )
}

export default nav

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/602d4e7a-c85a-4133-a8d0-d5c7c4f6158f)

**Comment by alex-krasikau** [comment, marked as answer]
Hey @MinalPatilRPA!

You just need to put a slash in front of the image path like this `<img src='/YTmp32-logo.png' className='w-full h-10' />`.

Please let me know if this helped you.
